### ISSUE TITLE
v3.25 Cannot modify server settings off of the global region (folia)

### ISSUE THREAD

**Issue by iLemon** [ChanceSD/PvPManager-Issues#184]
```
[12:37:44] [Server thread/ERROR]: Error occurred while enabling PvPManager v3.25 (Is it up to date?)
java.lang.IllegalStateException: Cannot modify server settings off of the global region
        at io.papermc.paper.threadedregions.RegionizedServer.ensureGlobalTickThread(RegionizedServer.java:106) ~[folia-1.20.1.jar:git-Folia-"a3b3914"]
        at org.bukkit.craftbukkit.v1_20_R1.CraftWorld.setGameRule(CraftWorld.java:2039) ~[folia-1.20.1.jar:git-Folia-"a3b3914"]
        at nc.gg.<init>(SourceFile:23) ~[PvPManager-3.25.jar:?]
        at me.NoChance.PvPManager.PvPManager.onEnable(SourceFile:1119) ~[PvPManager-3.25.jar:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:281) ~[folia-api-1.20.1-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:189) ~[folia-1.20.1.jar:git-Folia-"a3b3914"]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:104) ~[folia-1.20.1.jar:git-Folia-"a3b3914"]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:507) ~[folia-api-1.20.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_20_R1.CraftServer.enablePlugin(CraftServer.java:646) ~[folia-1.20.1.jar:git-Folia-"a3b3914"]
        at org.bukkit.craftbukkit.v1_20_R1.CraftServer.enablePlugins(CraftServer.java:557) ~[folia-1.20.1.jar:git-Folia-"a3b3914"]
        at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:679) ~[folia-1.20.1.jar:git-Folia-"a3b3914"]
        at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:463) ~[folia-1.20.1.jar:git-Folia-"a3b3914"]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:308) ~[folia-1.20.1.jar:git-Folia-"a3b3914"]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1188) ~[folia-1.20.1.jar:git-Folia-"a3b3914"]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:346) ~[folia-1.20.1.jar:git-Folia-"a3b3914"]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
[12:37:44] [Server thread/INFO]: [PvPManager] Disabling PvPManager v3.25
```

PvPManager 3.25
Folia 1.20.1

**Comment by ChanceSD** [comment]
Thank you, I believe I fixed this now, posted a build on the premium channel in discord, let me know if it helps
You didn't validate as a buyer there but I'm gonna trust you have the same nickname on spigot and discord and promoted you, so you should be able to see it

**Comment by ChanceSD** [comment]
Should be fixed in 3.26 which is now on Spigot
